tests/notebooks/training/training-integration.ipynb

```diff
@@ -6,9 +6,15 @@
    "source": [
     "# Test Training Operator Integration\n",
     "\n",
-    "This example notebook is loosely based on [this](https://github.com/kubeflow/training-operator/blob/master/sdk/python/examples/kubeflow-tfjob-sdk.ipynb) upstream example.\n",
+    "This example notebook is loosely based on the following upstream examples:\n",
+    "* [TFJob](https://github.com/kubeflow/training-operator/blob/964a6e836eedff11edfe79cc9f4e5b7c623cbe88/examples/tensorflow/image-classification/create-tfjob.ipynb)\n",
```

Comment:
I see we are using a commit hash instead of a branch, would that help if in the future they change something?

Reply:
Reason I did that is because if you try to follow the link that was there before, you 'll see that it doesn't work anymore and I couldn't find exactly the file that the initial example had been based on. I think it makes sense to keep the commit, thus when someone looks into updating those, they can just switch to the branch they 're interested in.

Reply:
Okay, that could help to keep things working. I think that once we upgrade to the training-operator 1.8 we could use the 1.8 tag, which is immutable.

No further action is needed then, thanks!